Scripts/script-DemistoDeleteIncident.yml

```diff
@@ -0,0 +1,35 @@
+commonfields:
+  id: DemistoDeleteIncident
+  version: -1
+name: DemistoDeleteIncident
+script: |-
+  var body = {
+      ids: [args.id],
```

Comment:
Maybe change art name or at least description if we can do it for a list as eell

Reply:
Don't like it. we can add another script for it. Deleting an incident is a very extreme case, so want to make sure it is differentiated. if you think we should add delete list - I will add.